packages/react-i18n/src/babel-plugin/shared.ts

```diff
@@ -29,7 +29,8 @@ export function findTranslationBuckets(rootDir) {
   // `find` is used here instead of Node's glob because it performs much faster
   // (20s vs 1s in web with ~750 translation folders and 21 langs)
   const files = execSync(
-    `find ${rootDir} -type d \\( -path ./node_modules -o -path ./build -o -path ./tmp -o -path ./.git -o -path ./public \\) -prune -o -name '*.json' -print | grep /${TRANSLATION_DIRECTORY_NAME}/`,
+    `find ${rootDir} -type d \\( -path ${rootDir}/node_modules -o -path ${rootDir}/build -o -path ${rootDir}/tmp -o -path ${rootDir}/.git -o -path ${rootDir}/public \\) -prune -o -name '*.json' -print | grep /${TRANSLATION_DIRECTORY_NAME}/`,
```

Comment:
The old version of this failed to ignore files because `rootDir` is a fully-qualified path (e.g., `/app/web/`), which `find` fails to match against relative paths like `./node_modules`.

The end result isn't so bad in CI because `build`, etc don't contain many matching paths.  It did match some node_modules paths, but libs like `@shopify/global-nav` never directly perform `useI18n()`, so the babel plugin parses them, but leaves them untouched.